scss/grid/_layout.scss

```diff
@@ -468,8 +468,13 @@
         }
         .k-header-column-menu {
             margin: 0;
-            margin-top: calc(#{$grid-form-component-offset} * -1);
-            margin-bottom: calc(#{$grid-form-component-offset} * -1);
+            margin-top: (-$button-padding-y);
+            margin-bottom: (-$button-padding-y);
```

Comment:
`$grid-form-component-offset` does perfect job of aligning components. Could you revise?